habitat-lab/habitat/core/utils.py

```diff
@@ -78,12 +78,15 @@ def not_none_validator(
 
 class Singleton(type):
     """
-    This metatclass creates Singleton objects by ensuring only one instance is created and any call is directed to that instance.
+    This metatclass creates Singleton objects by ensuring only one instance is created and any call is directed to that instance. TODO: DOUBLE CHECK THE FOLLOWING DESCRIPTION:  The mro() function and following dunders, EXCEPT __call__, are inherited from the the stdlib Python library, which defines the "type" class.
```

Comment:
Should I add this extra sentence into the Singleton class docstring?

Reply:
Sure, feel free to keep the extra details here.